### PR TITLE
Update codebase for obt7

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: src/js/forms.js [hunk 1]
```diff
@@ -99,7 +97,7 @@ class Forms {
 		this.validateInput(input);
 	}
 
-	handleClickEvent(event) {
+	static handleClickEvent(event) {
```
Comment (notlee) [inline src/js/forms.js]: >Expected 'this' to be used by class method 'handleClickEvent'.

@JakeChampion I followed your lead on making these static methods to get around the linting issue. It seems sensible but as it effects the api of an oForm instance it is technically a breaking change. I'm not worried in this case but thought if you hadn't realised it was worth mentioning incase it is an issue.

Comment (JakeChampion) [reply src/js/forms.js]: Very good point, we should instead disable the eslint error for these methods and create an issue to move these to static in the next major

File: test/forms.test.js [hunk 2]
```diff
@@ -158,6 +158,25 @@ describe("Forms", () => {
...
+			input.value = 'test';
+
+			formEl.addEventListener('submit', (event) => {
+				event.preventDefault();
+				throw new Error('Expected form to not be submitted');
+			}, false);
+
+			setTimeout(() => {
+				done();
+			}, 1000);
```
Comment (notlee) [inline test/forms.test.js]: This could probably be reduced from 1000 to speed up the Circle build?

**Comment by notlee** [comment]
Looks like pa11y tests are failing. Did they previously or do we have stricter obt7 settings?
`html > body > div:nth-child(36) > div > div > button`

**Comment by JakeChampion** [comment]
Not stricter settings, they failed for a while now on o-forms

**Pull request merged**